src/HeaderRow.tsx

```diff
@@ -60,7 +60,7 @@ function HeaderRow<R, SR, K extends React.Key>({
   selectCell,
   direction
 }: HeaderRowProps<R, SR, K>) {
-  const dragDropKey = useId();
+  const [draggedColumnKey, setDraggedColumnKey] = useState<string>();
```

Comment:
We can move this to the parent columns and reorder columns internally as user is dragging.  I will check